processor/schematransformerprocessor/factory_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schematransformerprocessor_test
```

Comment:
Yes, since the transformer is currently just an empty struct, I wasn't exactly sure how the transformer would look like so I didn't want to burden reviewers with more work than required.